server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -4894,6 +4894,7 @@ private void fillVMOrTemplateDetailOptions(final Map<String, List<String>> optio
             ApiConstants.BootMode.SECURE.toString()));
         options.put(VmDetailConstants.KEYBOARD, Arrays.asList("uk", "us", "jp", "fr"));
         options.put(VmDetailConstants.CPU_CORE_PER_SOCKET, Collections.emptyList());
+        options.put(VmDetailConstants.CPU_THREAD_PER_CORE, Collections.emptyList());
```

Comment:
@mlsorensen this detail is supported for KVM only with this PR, can move this to KVM if block?